pages/account.vue

```diff
@@ -181,6 +156,18 @@
             size="lg"
             block
           />
+
+          <div class="flex items-center justify-end w-full px-3">
+            <UButton
+              :label="$t('account_page_contact_support')"
+              variant="outline"
+              leading-icon="i-material-symbols-contact-support"
+              color="neutral"
+              :disabled="!isCrispLoaded"
+              size="lg"
+              @click="handleCustomerServiceLinkButtonClick"
+            />
+          </div>
```

Comment:
Why changing the button style?

Current
<img width="635" alt="Screenshot 2025-06-19 at 01 21 23" src="https://github.com/user-attachments/assets/1989d1f3-27d2-4758-9177-8ca4de10eed6" />

Change
<img width="572" alt="Screenshot 2025-06-19 at 01 22 07" src="https://github.com/user-attachments/assets/18432406-ecc8-493b-85c8-8b94b34f7ba6" />